### PR TITLE
[Enhancement] support query lake compaction profile in FE

### PR DESCRIPTION
**Pull request by starrocks-xupeng**
## What I'm doing:
ADD `profile` field

```
show proc '/compactions';

| ssb.lineorder.40462              | 30366 | 2024-09-10 19:48:23 | 2024-09-10 19:48:52 | 2024-09-10 19:48:52 | NULL  | {"sub_task_count":1,"read_local_sec":0,"read_local_mb":214,"read_remote_sec":5,"read_remote_mb":240,"in_queue_sec":0} |
| ssb.lineorder.40461              | 30367 | 2024-09-10 19:48:23 | 2024-09-10 19:48:50 | 2024-09-10 19:48:52 | NULL  | {"sub_task_count":1,"read_local_sec":0,"read_local_mb":215,"read_remote_sec":4,"read_remote_mb":239,"in_queue_sec":0} |
| ssb.lineorder.40463              | 30365 | 2024-09-10 19:48:22 | 2024-09-10 19:48:51 | 2024-09-10 19:48:51 | NULL  | {"sub_task_count":1,"read_local_sec":0,"read_local_mb":215,"read_remote_sec":5,"read_remote_mb":239,"in_queue_sec":0} |
```

```
select * from information_schema.be_cloud_native_compactions;

| 10001 |  30365 |     40472 |      16 |       0 |    1 | 2024-09-10 19:48:22 | NULL        |      100 |        | {"read_local_sec":0,"read_local_mb":215,"read_remote_sec":5,"read_remote_mb":239,"read_remote_count":241,"read_local_count":10195,"segment_init_sec":0,"column_iterator_init_sec":0,"in_queue_sec":0} |
| 10001 |  30366 |     40471 |      16 |       0 |    1 | 2024-09-10 19:48:23 | NULL        |       91 |        | {"read_local_sec":0,"read_local_mb":212,"read_remote_sec":5,"read_remote_mb":236,"read_remote_count":236,"read_local_count":9822,"segment_init_sec":0,"column_iterator_init_sec":0,"in_queue_sec":0}  |
| 10001 |  30367 |     40470 |      16 |       0 |    1 | 2024-09-10 19:48:23 | NULL        |      100 |        | {"read_local_sec":0,"read_local_mb":215,"read_remote_sec":4,"read_remote_mb":239,"read_remote_count":241,"read_local_count":10229,"segment_init_sec":0,"column_iterator_init_sec":0,"in_queue_sec":0} |
```

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by starrocks-xupeng** [comment]
@mergify rerun

File: fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionRecord.java [hunk 1]
```diff
@@ -66,4 +68,8 @@ public String getPartitionName() {
     public Optional<String> getErrorMessage() {
         return errorMessage != null ? Optional.of(errorMessage) : Optional.empty();
     }
+
+    public Optional<String> getExecutionProfile() {
+        return (executionProfile != null && !executionProfile.isEmpty()) ? Optional.of(executionProfile) : Optional.empty();
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionRecord.java:73]:
```suggestion
        return Strings.isNullOrEmpty(executionProfile) ? Optional.empty() : Optional.of(executionProfile);
```

**Review by murphyatwork — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionRecord.java (hunk 1 above)
Comment (starrocks-xupeng) [reply fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionRecord.java]: done

**Review by murphyatwork — APPROVED** [review]

**Review by kevincai — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Review by wangruin — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by starrocks-xupeng** [comment]
https://github.com/StarRocks/starrocks/pull/51275, branch-3.2

**Pull request merged**